pkg/capi/util/validators.ts

```diff
@@ -121,3 +122,22 @@ const stringFormatValidators = function(t: Translation, { key = 'Value' }: Valid
 };
 
 export const isDefined = (val: any) => (val || val === false) || !isEmpty(val);
+
+export const versionTest = function(t: Translation, type: string): RegExp {
+  let ending = '';
+
+  if (CP_VERSIONS[type as keyof typeof CP_VERSIONS]) {
+    ending = `\\+(${ CP_VERSIONS[type as keyof typeof CP_VERSIONS].join('|') })`;
+  }
+
+  return new RegExp(`^v(\\d+.){2}\\d+${ ending }$`);
```

Comment:
Not sure about this regex, nothing I enter is valid. Can you give an example of a version that works for you? Either way I think the error message needs to be more descriptive

Reply:
I don't think we can make it more descriptive because different control planes will have different formats

Reply:
I think it's doable because we know all the valid formats in the validator, it's just a more complex message to construct. We can revisit this when we go back to refine this feature though